### PR TITLE
feat: add add breadcrumb content component

### PR DESCRIPTION
**Pull request by RobbeBierebeeck**
This branch has a dependency on the Breadcrumb component in the react-components package

### CODE REVIEW

File: ui/src/react-admin/modules/content-page/components/blocks/BlockBreadcrumbs/BlockBreadcrumbs.editorconfig.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+								'INTERNAL_LINK',
+								'EXTERNAL_LINK',
+							] as ContentPickerType[],
+						},
+					},
+				},
+				type: 'fieldGroup',
+				repeat: {
+					defaultState: DEFAULT_BUTTON_PROPS,
+					addButtonLabel: 'voeg object toe',
```
Comment (bertyhell) [inline ui/src/react-admin/modules/content-page/components/blocks/BlockBreadcrumbs/BlockBreadcrumbs.editorconfig.ts:71]: tText wrapper

File: ui/src/react-admin/modules/content-page/components/blocks/BlockBreadcrumbs/BlockBreadcrumbs.editorconfig.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+					deleteButtonLabel: 'verwijder object',
```
Comment (bertyhell) [inline ui/src/react-admin/modules/content-page/components/blocks/BlockBreadcrumbs/BlockBreadcrumbs.editorconfig.ts:72]: tText wrapper

File: ui/src/react-admin/modules/content-page/components/blocks/BlockBreadcrumbs/BlockBreadcrumbs.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import { Icon } from '~shared/components';
+import { Link } from '~shared/components/Link';
+
+export const BlockBreadcrumbs: FC<BlockBreadcrumbsProps> = ({ foregroundColor, elements }) => {
+	useEffect(() => {
+		console.log(elements);
+	}, [elements]);
+	return (
+		<Breadcrumbs
+			icon={<Icon name={'angleRight'} />}
```
Comment (bertyhell) [inline ui/src/react-admin/modules/content-page/components/blocks/BlockBreadcrumbs/BlockBreadcrumbs.tsx:13]:
```suggestion
			icon={<Icon name="angleRight" />}
```

File: ui/src/react-admin/modules/content-page/types/content-block.types.ts [hunk 4]
```diff
@@ -156,6 +156,7 @@ export enum ContentBlockType {
 	HetArchiefHeaderSearch = 'HETARCHIEF__HEADER_SEARCH',
 	OverviewNewspaperTitles = 'OVERVIEW_NEWSPAPER_TITLES',
 	ContentEncloseGrid = 'CONTENT_ENCLOSE_GRID',
+	Breadcrumbs = 'BREADCRUMBS',
```
Comment (bertyhell) [inline ui/src/react-admin/modules/content-page/types/content-block.types.ts:159]: does this BREADCRUMBS block also have to be added to the hasura using a migration? in the lookup block types table?

**Review by bertyhell — APPROVED** [review]

File: ui/src/react-admin/modules/content-page/types/content-block.types.ts (hunk 4 above)
Comment (RobbeBierebeeck) [reply ui/src/react-admin/modules/content-page/types/content-block.types.ts:159]: No it is already in the database

**Review by ian-emsens-shd — APPROVED** [review]

**Pull request merged**